src/main/java/org/jabref/gui/preferences/PreferencesDialog.java

```diff
@@ -62,6 +64,10 @@ public PreferencesDialog(JabRefFrame parent, TaskExecutor taskExecutor) {
         setTitle(Localization.lang("JabRef preferences"));
         getDialogPane().getScene().getStylesheets().add(this.getClass().getResource("PreferencesDialog.css").toExternalForm());
 
+        ViewLoader.view(this)
```

Comment:
now the above statement loading the `css` file should be superfluous.

Reply:
Nope, the CSS specified some other preferences stuff, e.g. the side menu